src/custom/theme/baseTheme.tsx

```diff
@@ -53,6 +53,9 @@ export function colors(darkMode: boolean): Colors {
     blueShade2: '#011e34',
     border: darkMode ? '#3a3b5a' : 'rgb(58 59 90 / 10%)',
     border2: darkMode ? '#254F83' : '#afcbda',
+    cardBorder: darkMode ? '#021E34' : 'rgba(255, 255, 255, 0.5)',
+    cardShadow21: darkMode ? '#4C7487' : '#FFFFFF',
+    cardShadow22: darkMode ? 'rgba(1, 10, 16, 0.15)' : 'rgba(11, 37, 53, 0.93)',
```

Comment:
Any reason for using `21` then `22`?
Maybe you meant to have `1` and `2` ? Or no index and `2`?

Reply:
yes, at first i thought about using indexes and forgot to remove it later. 2-index removed